### PR TITLE
Trim down the list of directories watched by CompileDaemon

### PR DESCRIPTION
**Pull request by bl-nero**
This makes it faster and consumes fewer system resources.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -299,7 +299,18 @@ $(BUILDDIR)/teleport: ensure-webassets bpf-bytecode rdpclient
 TELEPORT_ARGS ?= start
 .PHONY: teleport-hot-reload
 teleport-hot-reload:
-	CompileDaemon --graceful-kill=true --exclude-dir=".git" --exclude-dir="node_modules" --build="make $(BUILDDIR)/teleport" --command="$(BUILDDIR)/teleport $(TELEPORT_ARGS)"
+	CompileDaemon \
+	  --graceful-kill=true \
+		--exclude="node_modules" \
```
Comment (AntonAM) [inline Makefile:304]: This one shouldn't be here?

**Review by AntonAM — APPROVED** [review]

**Comment by AntonAM** [comment]
Also, it seems there's some indentation issue

File: Makefile (hunk 1 above)
Comment (bl-nero) [reply Makefile:304]: Why should it? The purpose of CompileDaemon is to monitor and reload the Go files, not the TypeScript ones. (For TypeScript, we have Vite dev server).

Comment (AntonAM) [reply Makefile:304]: We already have `--exclude-dir="node_modules" \` a bit down and here it's `exclude` (not `exclude-dir`) which is supposed to be used with files, not directories.

**Comment by bl-nero** [comment]
> Also, it seems there's some indentation issue

Corrected.

File: Makefile (hunk 1 above)
Comment (bl-nero) [reply Makefile:304]: Yes, I've just noticed it and deleted my comment, but you were faster :D

**Review by AntonAM — APPROVED** [review]

**Review by marcoandredinis — APPROVED** [review]

**Review by tigrato — APPROVED** [review]

**Pull request merged**